### PR TITLE
Fix and Update jSON-RPC documentation

### PR DESCRIPTION
**Pull request by ssinghaldev**
## Motivation

Fixing https://github.com/libra/libra/issues/4160
Adding doc for another endpoint

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

N/A

## Related PRs
N/A

### CODE REVIEW

File: json-rpc/json-rpc-spec.md [hunk 1]
```diff
@@ -633,11 +713,11 @@ A Libra account.
    </td>
   </tr>
   <tr>
-   <td>balance
+   <td>balances
    </td>
-   <td><a href="#amount---type">Amount</a>
+   <td><a href="#amount---type">Array of Amount objects </a>
```
Comment (thefallentree) [inline json-rpc/json-rpc-spec.md:718]: the <a> should still be on Amount

File: json-rpc/json-rpc-spec.md [hunk 2]
```diff
@@ -575,6 +575,86 @@ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","metho
```
Comment (thefallentree) [inline json-rpc/json-rpc-spec.md:577]: too many extra whitespace?

File: json-rpc/json-rpc-spec.md (hunk 1 above)
Comment (ssinghaldev) [reply json-rpc/json-rpc-spec.md:718]: U mean hyper-link should be only on Amount but the txt can be "Array of Amount objects" Or it should just point to Amount?

File: json-rpc/json-rpc-spec.md (hunk 2 above)
Comment (ssinghaldev) [reply json-rpc/json-rpc-spec.md:577]: Seems to follow the existing spaces b/w API description. Even the rendered view looks ok

File: json-rpc/json-rpc-spec.md (hunk 1 above)
Comment (thefallentree) [reply json-rpc/json-rpc-spec.md:718]: the hyperlink should only contain "Amount" as that is the type

Comment (ssinghaldev) [reply json-rpc/json-rpc-spec.md]: Changed it back to Amount

Comment (thefallentree) [reply json-rpc/json-rpc-spec.md]: no man, i meant it should be 
```
   <td><span>Array of <a href="#amount---type">Amount</a> objects </span>
```

Comment (ssinghaldev) [reply json-rpc/json-rpc-spec.md]: My bad :-(. Fixed it!

**Pull request closed without merging**